### PR TITLE
chore: Add robust example in storybook for ToolbarDropdown button

### PR DESCRIPTION
**Pull request by NehaAhujaa**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/2730

## Release Category
Documentation

---

## Checklist

- [x] MDX documentation adheres to Canvas Kit's [Documentation Guidelines](https://workday.github.io/canvas-kit/?path=/docs/guides-documentation-guidelines--page)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [x] PR title is short and descriptive
- [x] PR summary describes the change (Fixes/Resolves linked correctly)
- [na] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [na] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

## Screenshots or GIFs (if applicable)

Before:

Screenshot 2024-06-14 at 14 58 08

After:

Screenshot 2024-06-17 at 14 13 34

### CODE REVIEW

**Comment by mannycarrera4** [comment]
Thank you Neha for making out docs better! Small suggestion, maybe use an icon and text for the menu list that might reflect a more real life example. If you use `alignLeftIcon` and then the options can be text that says let align, center align, right align.

**Comment by NehaAhujaa** [comment]
> Thank you Neha for making our docs better! Small suggestion, maybe use an icon and text for the menu list that might reflect a more real life example. If you use `alignLeftIcon` and then the options can be text that says let align, center align, right align.

@mannycarrera4 Thank you, Manuel! Incorporated the suggested changes to make example meaningful.

File: modules/react/button/stories/stories_ToolbarButtons.tsx [hunk 1]
```diff
@@ -43,4 +44,50 @@ storiesOf('Components/Buttons/Toolbar', module)
...
+            onClick={() => {
+              console.log('Expand icon clicked');
+            }}
+          ></Menu.Target>
+        </Tooltip>
+        <Menu.Popper>
+          <Menu.Card>
+            <Menu.List>
+              <Menu.Item
+                aria-label="Expand All"
```
Comment (mannycarrera4) [inline modules/react/button/stories/stories_ToolbarButtons.tsx:65]: aria label is not needed unless we have an icon.

Comment (NehaAhujaa) [reply modules/react/button/stories/stories_ToolbarButtons.tsx:65]: oh yes, makes sense! My bad there!

**Review by mannycarrera4 — APPROVED** [review]
Thanks for improving our docs!

**Pull request merged**